dist-git.spec

```diff
@@ -26,18 +26,26 @@ Requires:       git
 Requires:       git-daemon
 Requires:       mod_ssl
 Requires:       crudini
-Requires:       moreutils
+# Requires:       moreutils
```

Comment:
I'd remove it from here. There is already a nice comment saying that we would like to have it in global Requires as soon as possible, so this line seems redundant.